.zuul.yaml

```diff
@@ -0,0 +1,77 @@
+- job:
+    name: rust-base
```

Comment:
just realized that this job is duplicated with https://github.com/dtantsur/rust-openstack/blob/master/zuul.d/jobs.yaml#L2

This is not allowed in openlab by default. (https://zuul-ci.org/docs/zuul/reference/tenants.html#attr-tenant.untrusted-projects.%3Cproject%3E.shadow  we disabled it) 

Can you rename it?

Reply:
Can I just reuse the job then? Or do I have to redefine it with a new name?

Reply:
I think you can, if they are the same func

Reply:
W00t, it works, many thanks for your help!